client/state/data-layer/wpcom/sites/memberships/index.js

```diff
@@ -57,6 +55,23 @@ export const handleMembershipsList = dispatchRequestEx( {
 	onError: noop,
 } );
 
+export const handleSubscribedMembershipsList = dispatchRequestEx( {
+	fetch: action =>
+		http(
+			{
+				method: 'GET',
+				path: '/me/memberships/subscriptions',
+			},
+			action
+		),
+	onSuccess: ( {}, { subscriptions, total } ) => ( {
```

Comment:
what is the `{}` for in the first argument? if it's to avoid any linting just ignore the linter - we should have a name like `action` here

Reply:
Done!